src/main/java/com/example/bulletinboardapp/controller/AdController.java

```diff
@@ -0,0 +1,57 @@
+package com.example.bulletinboardapp.controller;
+
+import com.example.bulletinboardapp.entity.Ad;
+import com.example.bulletinboardapp.dto.Ads;
+import com.example.bulletinboardapp.dto.CreateOrUpdateAd;
+import com.example.bulletinboardapp.dto.ExtendedAd;
+import io.swagger.annotations.Api;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+@Api(tags = "Объявления")
+@Slf4j
+@CrossOrigin(value = "http://localhost:3000")
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/ads")
+public class AdController {
+
+    @GetMapping
+    public Ads getAllAds() {
+        return new Ads();
+    }
+
+    @PostMapping
+    public Ad create(@RequestBody Ad ad,
+                     @RequestBody MultipartFile image) {
+        return new Ad();
+    }
+
+    @GetMapping("/{id}")
+    public ExtendedAd getAdById(@PathVariable Integer id) {
+        return new ExtendedAd();
+    }
+
+    @DeleteMapping("/{id}")
+    public void delete(@PathVariable Integer id) {
+
+    }
+
+    @PatchMapping("/{id}")
+    public CreateOrUpdateAd updateAd(@PathVariable Integer id) {
+        return new CreateOrUpdateAd();
+    }
+
+    @GetMapping("/me")
+    public Ads getUsersAd() {
+        return new Ads();
+    }
+
+    @PatchMapping("/{id}/image")
```

Comment:
```suggestion
    @PatchMapping(value = "/{id}/image", consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
```
вот так должно быть, иначе работать не будет